pkg/plugins/policies/meshhttproute/api/v1alpha1/zz_generated.resource.go

```diff
@@ -0,0 +1,134 @@
+
+	"k8s.io/kube-openapi/pkg/validation/spec"
+	"sigs.k8s.io/yaml"
+
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+//go:embed schema.yaml
+var rawSchema []byte
+var schema = spec.Schema{}
+
+func init() {
+	if err := yaml.Unmarshal(rawSchema, &schema); err != nil {
+		panic(err)
+	}
+}
+
+const (
+	MeshHTTPRouteType model.ResourceType = "MeshHTTPRoute"
+)
+
+var _ model.Resource = &MeshHTTPRouteResource{}
+
+type MeshHTTPRouteResource struct {
+	Meta model.ResourceMeta
+	Spec *MeshHTTPRoute
+}
+
+func NewMeshHTTPRouteResource() *MeshHTTPRouteResource {
+	return &MeshHTTPRouteResource{
+		Spec: &MeshHTTPRoute{},
+	}
+}
+
+func (t *MeshHTTPRouteResource) GetMeta() model.ResourceMeta {
+	return t.Meta
+}
+
+func (t *MeshHTTPRouteResource) SetMeta(m model.ResourceMeta) {
+	t.Meta = m
+}
+
+func (t *MeshHTTPRouteResource) GetSpec() model.ResourceSpec {
+	return t.Spec
+}
+
+func (t *MeshHTTPRouteResource) SetSpec(spec model.ResourceSpec) error {
+	protoType, ok := spec.(*MeshHTTPRoute)
+	if !ok {
+		return fmt.Errorf("invalid type %T for Spec", spec)
+	} else {
+		if protoType == nil {
+			t.Spec = &MeshHTTPRoute{}
+		} else {
+			t.Spec = protoType
+		}
+		return nil
+	}
+}
+
+func (t *MeshHTTPRouteResource) Descriptor() model.ResourceTypeDescriptor {
+	return MeshHTTPRouteResourceTypeDescriptor
+}
+
+func (t *MeshHTTPRouteResource) Validate() error {
+	if v, ok := interface{}(t).(interface{ validate() error }); !ok {
+		return nil
+	} else {
+		return v.validate()
+	}
+}
+
+var _ model.ResourceList = &MeshHTTPRouteResourceList{}
+
+type MeshHTTPRouteResourceList struct {
+	Items      []*MeshHTTPRouteResource
+	Pagination model.Pagination
+}
+
+func (l *MeshHTTPRouteResourceList) GetItems() []model.Resource {
+	res := make([]model.Resource, len(l.Items))
+	for i, elem := range l.Items {
+		res[i] = elem
+	}
+	return res
+}
+
+func (l *MeshHTTPRouteResourceList) GetItemType() model.ResourceType {
+	return MeshHTTPRouteType
+}
+
+func (l *MeshHTTPRouteResourceList) NewItem() model.Resource {
+	return NewMeshHTTPRouteResource()
+}
+
+func (l *MeshHTTPRouteResourceList) AddItem(r model.Resource) error {
+	if trr, ok := r.(*MeshHTTPRouteResource); ok {
+		l.Items = append(l.Items, trr)
+		return nil
+	} else {
+		return model.ErrorInvalidItemType((*MeshHTTPRouteResource)(nil), r)
+	}
+}
+
+func (l *MeshHTTPRouteResourceList) GetPagination() *model.Pagination {
+	return &l.Pagination
+}
+
+var MeshHTTPRouteResourceTypeDescriptor = model.ResourceTypeDescriptor{
+	Name:                MeshHTTPRouteType,
+	Resource:            NewMeshHTTPRouteResource(),
+	ResourceList:        &MeshHTTPRouteResourceList{},
+	Scope:               model.ScopeMesh,
+	KDSFlags:            model.FromGlobalToZone,
+	WsPath:              "meshhttproutes",
+	KumactlArg:          "meshhttproute",
+	KumactlListArg:      "meshhttproutes",
+	AllowToInspect:      true,
+	IsPolicy:            true,
+	IsExperimental:      false,
+	SingularDisplayName: "Mesh H T T P Route",
```

Comment:
This doesn't look right :D

Reply:
https://github.com/kumahq/kuma/pull/5595 @lobkovilya